apps/src/studio/studio.js

```diff
@@ -187,7 +189,16 @@ var consoleLogger = null;
 
 function loadLevel() {
   // Load maps.
-  Studio.map = level.map;
+  Studio.map = level.map.map(function (row) {
+    return row.map(function (cell) {
+      // Each cell should be either an integer (in which case we are
+      // dealing with the legacy format and should treat that value as
+      // the tileType for the cell) or an object (in which case we are
+      // dealing with the new format and should treat that value as a
+      // serialization of the cell).
+      return isNaN(parseInt(cell)) ? studioCell.deserialize(cell) : new studioCell(cell);
+    });
+  });
```

Comment:
oh, hell yeah! I originally wrote this block before the switchover. This PR is probably half ES5 and half ES6 :/